### PR TITLE
Added footer section in Country details app

### PR DESCRIPTION
**Pull request by Kedarnath-Rothe**
### ✅ Check List (Check all the applicable boxes) 

- [ ] My code doesn't break any part of the project
- [ ] This PR does not contain plagiarized content.
- [ ] My Addition/Changes works properly and matches the overall repo pattern.
- [ ] The title of my pull request is a short description of the requested changes.

### 📷 Screenshots

#Fixed issue #1039
Footer

### CODE REVIEW

File: 50 - Country Detail/index.html [hunk 1]
```diff
@@ -45,12 +46,21 @@ <h1>Country Details App</h1>
...
           <div class="col-md-4">&lt;/> with ❤ by <a href="https://github.com/TushitaParashar" style="text-decoration:none" >Tushita Parashar</a></div>
-        </div>
+        </div> -->
 
-    </div>
+        
+      </div>
+      <footer>
+        <div>&lt;/&gt; with ❤ by <a href="https://github.com/TushitaParashar" target="_blank">Tushita Parashar</a></div> <br>
+        <div class="subfooter" >
```
Comment (swapnilsparsh) [inline 50 - Country Detail/index.html:58]: Remove this subfooter div. Dont add extra code.

Comment (Kedarnath-Rothe) [reply 50 - Country Detail/index.html:58]: ok

**Comment by Kedarnath-Rothe** [comment]
Done
Footer

**Comment by Kedarnath-Rothe** [comment]
Please see my PR... Have i have Removed  subfooter fron HTML

**Pull request merged**